openedx/core/djangoapps/persistenttask/migrations/0001_initial.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='FailedTask',
+            fields=[
+                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
+                ('task_name', models.CharField(max_length=255, db_index=True)),
+                ('task_id', models.CharField(max_length=255)),
+                ('argstring', models.TextField(blank=True)),
+                ('kwargstring', models.TextField(blank=True)),
+                ('exc', models.CharField(max_length=255)),
```

Comment:
I want to keep the names aligned with the names used by celery.  Another name might be "better" on its own, but it means that readers have to deal with multiple naming conventions as they track things through celery into our table.